.circleci/config.yml

```diff
@@ -154,11 +154,11 @@ jobs:
           name: "Prepare JS assets" # Compile our final, production-ready JavaScript.
           command: yarn webpack-prod
       - run:
-          name: "Persist JS assets" # Move our JS into our workspace.
+          name: "Persist JS assets" # Move our JS+CSS into our workspace.
           command: |
             set -exu
             mkdir -p /tmp/workspace/js
-            mv jekyll/assets/js/*.bundle.js /tmp/workspace/js/
+            mv jekyll/assets/js/* /tmp/workspace/js/
```

Comment:
Does this mean we are putting the `app.css` file inside the `js/` directory?

Reply:
Yep, I have two questions about this:

- how do you feel about `styles` being under `src-js`? I could potentially put it outside src-js but that seems like it's not idiomatic to import from outside the src-js folder? (or perhaps we rename src-js to `src`- and then have a top level `js` and `styles` folder?) 
- The above is why webpack exports the final css file adjacent to the bundled js, I could move it out of the js folder at pipeline build time _or_ see if webpack can configure this a bit better.

Reply:
Yeah, option 1 seems to make more sense to me! I'm also okay if we want to output files in a directory called `bundles/` (or a better name) and have inside the `app.js` & `app.css`.